### ISSUE TITLE
June Meetup

### ISSUE THREAD

**Issue by mattparrilla** [burlingtonjs/burlingtonjs#8]
- [x] Lock in Speaker 
- [x] Lock in date with speaker 
- [x] Lock in date with Venue
- [x] create meetup
- [x] market meetup (eg twitter) 
- [x] see if speaker will sponsor (?)
- [x] Order Pizza

**Comment by donnie-coleman** [comment]
Going to reach out to John Need. If that doesn't work, going to reach out to Dave Davis. Hoping they're agree to June 6th. I've already penciled us in on that date in conjunction with Python. https://docs.google.com/document/d/1Tt_g3aa0Wn6vGD7tc7jXxhZDIbtMKzSw2o25NlC-ZXI/edit#

**Comment by jhoguet** [comment]
I ran into John Need today and he said as long as the date works he would be up for doing a talk on react-router

**Comment by donnie-coleman** [comment]
Do I still need to see if the speaker will sponsor?

**Comment by jhoguet** [comment]
Good question. I think we should be transparent that we have a sponsor for this meetup, but we should leverage the opportunity to ask them to sponsor for a future meetup. If they do sponsor, we should give them credit at the meetup that they are sponsoring a future meetup (so they still get some air time coinciding with a speaker from their company)

https://donorbox.org/burlington-js-pizza-q2

**Comment by jhoguet** [comment]
@mattparrilla I have a checkbook for the Burlington Code account now... if that helps...

**Comment by donnie-coleman** [comment]
@jhoguet I think that will help me for pizza for June. We have to pick it up ourselves, right @mattparrilla? Also, what's our formula for # of guests vs. # of pizzas?

**Comment by mattparrilla** [comment]
@donnietron, yes. For the Leonardos non-profit deal we have to pick it up ourselves.

Also, I updated the pizza doc with the formula: https://docs.google.com/document/d/1Qpfft0cuoc03AJjTbfg3PcnlXm28YDyPhRwE8nBonyo/edit?usp=sharing

TLDR: # of attendees / 6

**Comment by donnie-coleman** [comment]
@jhoguet could you tell me the current sponsors on the donorbox page so I can give them a shoutout (i'll probably ask you in person, too)

**Comment by jhoguet** [comment]
https://docs.google.com/document/d/1oNbyLmCjDcz-ElY1t1OMFYaTlP9AKiOT-7TadUmc6KU/edit
